opentelemetry-sdk/src/trace/span_processor.rs

```diff
@@ -149,14 +149,10 @@ impl SpanProcessor for SimpleSpanProcessor {
     }
 
     fn shutdown(&self) -> TraceResult<()> {
```

Comment:
I think we should (in another issue!) chase this distinction between `TraceError` and `ShutdownError` further down the rabbit hole. For the same reasons I don't think it makes sense to use an über-error in the exporter trait ...